packages/react-core/src/components/Tabs/Tab.tsx

```diff
@@ -7,7 +7,7 @@ export interface TabProps extends Omit<React.HTMLProps<HTMLAnchorElement | HTMLB
   className?: string;
   /** URL associated with the Tab. A Tab with an href will render as an <a> instead of a <button>. A Tab inside a <Tabs variant="nav"> should have an href. */
   href?: string;
-  /** Tab title */
+  /** Content rendered in the tab title */
   title: React.ReactNode;
```

Comment:
It appears that `TabTitleText` is required for proper styling. Shouldn't the type reflect that?

Reply:
Can you also add that to your PR description? Looks like the components to use are not mentioned